src/plugins/ConsolidatePlugin.ts

```diff
@@ -54,8 +58,9 @@ export class ConsolidatePluginClass implements Plugin {
       file.contents = await consolidate[this.engine].render(file.contents, {
         cache: false,
         filename: 'base',
-        basedir: file.context.homeDir,
-        includeDir: file.context.homeDir
+        basedir: this.basedir || file.context.homeDir,
```

Comment:
where do you assign `basedir`? it's not in the constructor

Reply:
Looks like its optional so should be fine. I am fussy, could we capitalise the D in basedir as well for consistency 😁

Reply:
Let's capitlise it ! 😄